modules/imgproc/test/test_goodfeaturetotrack.cpp

```diff
@@ -488,6 +502,19 @@ int CV_GoodFeatureToTTest::validate_test_results( int test_case_idx )
         ts->set_failed_test_info(cvtest::TS::OK);
     }
 
+    e = cv::norm(cornersQuality, RefcornersQuality, NORM_RELATIVE | NORM_INF);
+
+    if (e > eps)
+    {
+        ts->printf(cvtest::TS::CONSOLE, "actual error: %g, expected: %g", e, eps);
+        ts->set_failed_test_info(cvtest::TS::FAIL_BAD_ACCURACY);
```

Comment:
Prefer to use GTest's `EXPECT_LE(e, eps);` instead of legacy "ts->printf()".
It would should failed line at least.

Reply:
Fixed, but unfortunately the test itself does not extend GTest so semantics of EXPECT_LE looks oddly.